warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -187,15 +187,19 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
         if ip is None:
             ip = data_Utils.getSystemData(self.datafile, system_name, "ip")
         if remote is None:
-            remote = data_Utils.getSystemData(self.datafile, system_name,
-                                              "remote")
-
-        webdriver_remote_url = ip if str(remote).strip().lower() == "yes"\
-            else False
-
-        system = xml_Utils.getElementWithTagAttribValueMatch(self.datafile,
-                                                             "system",
-                                                             "name",
+            remote = data_Utils.getSystemData(self.datafile, system_name, "remote")
+        if binary is None:
+            binary = data_Utils.getSystemData(self.datafile, system_name, "binary")
```

Comment:
binary, gecko_path. proxy_ip and proxy_port aren't in system
It's inside browser level